docs/docs/examples/url-shortener.md

```diff
@@ -187,14 +187,14 @@ class ShortenerConfig:
             description="Creates a new short URL entry in DynamoDB mapping to the original url",
             directory="shortener",
             environment={
-                "URLS_TABLE_NAME": services.dynamo_db.urls_table.table_name,
+                "URLS_TABLE_NAME": services.dynamodb.urls_table.table_name,
                 "BASE_URL": context.resources["base_url"],
             },
         )
 
         services.api_gateway.create_endpoint("POST", "/urls", function, public=True)
 
-        services.dynamo_db.urls_table.grant_write_data(function)
```

Comment:
We need to alter here to use the method from the dynamodb class as well

Reply:
I will change